src/main/java/teammates/ui/request/FeedbackSessionUpdateRequest.java

```diff
@@ -1,8 +1,54 @@
 package teammates.ui.request;
 
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+
 /**
  * The update request of a feedback session.
  */
 public class FeedbackSessionUpdateRequest extends FeedbackSessionBasicRequest {
 
+    private Map<String, Long> studentDeadlines;
+    private Map<String, Long> instructorDeadlines;
+
+    private boolean isGoingToNotifyAboutDeadlines;
```

Comment:
Would you need another boolean? I know Samuel's email has a revoke and update version. From this endpoint, I don't think we're able to differentiate

Reply:
@jayasting98, sorry I think I made a mistake suggesting that we include a boolean `notifyUsersOfDeadlines` in the Update Request. I now think it might be more "RESTful" to include it as a query param instead.

Reply:
@FergusMok, no need! I can differentiate on my side.

Reply:
Yes, I was wondering about this as well. It felt strange to include it since it was not really a property of the entity somehow. I think you're right about REST; that's probably it. I think it does fit better as a query param. I will change it.